setup.py

```diff
@@ -0,0 +1,25 @@
+from setuptools import setup, find_packages
+
+setup(
+    name = "p4runtime",
+    version = "1.2.0",
+    packages = find_packages("py"),
+    install_requires = [
+        "protobuf>=3.5.0",
```

Comment:
nit: inconsistent spacing around " >= " compared to the other dependencies below